metadata.tex

```diff
@@ -76,6 +76,21 @@
 \newtoggle{noartbackground}
 \newtoggle{githubbuild}
 
+% Control addition of certain pages in the printable version.
+% Useful to make the printable version have correct number of pages.
+\newtoggle{printable_quote_separate_page}
+\newtoggle{printable_notes_page}
+\togglefalse{printable_quote_separate_page}
+\toggletrue{printable_notes_page}
```

Comment:
I hope it's not going to get out of hand.

Reply:
Me too